react/features/chat/components/web/Chat.js

```diff
@@ -88,6 +87,37 @@ class Chat extends AbstractChat<Props> {
         );
     }
 
+    /**
+     * Iterates over all the messages and creates nested arrays which hold
+     * consecutive messages sent be the same participant.
+     *
+     * @private
+     * @returns {Array<Array<Object>>}
+     */
+    _getMessagesGroupedBySender() {
```

Comment:
I think we'll probably need this on mobile too. How much effort do you think would it take to make it generic and move it to the abstract class?

Reply:
I think I moved it in a follow up PR where mobile uses it. At least I hope I moved it then.